modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -319,21 +374,21 @@ The index name can also be given as a parameter, `DROP INDEX $name`.
 == Set a vector property on a node
 
 Valid vectors for use in the index must have components finitely representable in {ieee-754} _single_ precision.
-They are represented as properties on nodes with the type `LIST<FLOAT>`.
+They are represented as properties on nodes with the type `LIST<INTEGER | FLOAT>`.
 As of Neo4j 5.13, you can set a vector property using the {link-procedures-reference}#procedure_db_create_setNodeVectorProperty[`db.create.setNodeVectorProperty`] procedure.
 It validates the input and sets the property as an array of {ieee-754} single precision values.
 This beta procedure replaces {link-procedures-reference}#procedure_db_create_setVectorProperty[`db.create.setVectorProperty`].
 
 .Signature for `db.create.setNodeVectorProperty`
 [source,syntax]
 ----
-db.create.setNodeVectorProperty(node :: NODE, key :: STRING, vector :: LIST<FLOAT>)
+db.create.setNodeVectorProperty(node :: NODE, key :: STRING, vector :: LIST<INTEGER | FLOAT>)
```

Comment:
I guess we still need to document the relationship set property procedure?

Reply:
Do we want an example of using it as well or are we happy with just listing it? (might be a question for docs team)